Dockerfile

```diff
@@ -1,26 +1,37 @@
-FROM php:7.3-cli
+FROM php:7-cli
```

Comment:
locknul bych to na 7.4

Reply:
minule mi Tomas pisal, ze to je lepsie takto, .... idem najst ten koment ... mne je to v principe jedno, ale tak je to aj v `db-extractor-common` tak som to chcel zachovat jednotne.

Reply:
aj v commonu jsem to teda přehlídl... z principu mi příjde lepší to locknout na verzi, protože až vydají 7.5/6/7... tak se to na ní bude updatovat a to imho úplně nechceme, protože např přechod 7.3->7.4 se taky úplně neobešel bez úprav

Reply:
upravil bych to teda ještě v commonu

Reply:
jj. mas pravdu. zmylil som sa, a zle som si to zapamatal, tam sa riesila ina situacia ... teda locknem to tu aj v `db-extractor-config`.

Reply:
locknute v https://github.com/keboola/db-extractor-table-format/pull/10/commits/5816a7022c54d51fdd7c8646d135198446cf6f2a

a aj v `db-extractor-config` https://github.com/keboola/db-extractor-config/pull/6/commits/ae43ffe69afa908e051a72fe4d1769816eb1e61e